debezium-core/src/main/java/io/debezium/transforms/TimezoneConverter.java

```diff
@@ -0,0 +1,388 @@
+                record.timestamp(),
+                record.headers());
+    }
+
+    @Override
+    public void configure(Map<String, ?> configs) {
+        final Configuration config = Configuration.from(configs);
+        SmtManager<R> smtManager = new SmtManager<>(config);
+        smtManager.validate(config, Field.setOf(CONVERTED_TIMEZONE, INCLUDE_LIST, EXCLUDE_LIST));
+
+        convertedTimezone = config.getString(CONVERTED_TIMEZONE);
+        includeList = config.getList(INCLUDE_LIST);
+        excludeList = config.getList(EXCLUDE_LIST);
+
+        validateConfiguration();
+
+        if (!excludeList.isEmpty()) {
+            collectTablesAndTopics(excludeList);
+        }
+        else if (!includeList.isEmpty()) {
+            collectTablesAndTopics(includeList);
+        }
+    }
+
+    private void collectTablesAndTopics(List<String> list) {
+        String commonPrefix = null;
+        for (String item : list) {
+            FieldItem parseItem = parseItem(item);
+            String prefix = parseItem.prefix;
+
+            if (prefix != null) {
+                commonPrefix = prefix;
+            }
+
+            String matchName = parseItem.getMatchName();
+            if (commonPrefix != null && (!cache.containsKey(matchName + ":before") || !cache.containsKey(matchName + ":after"))) {
+                if (Objects.equals(commonPrefix, TOPIC) || Objects.equals(commonPrefix, SOURCE)) {
+                    topicsAndTables.add(matchName);
+                }
+            }
+        }
+    }
+
+    private void validateConfiguration() {
+        if (!includeList.isEmpty() && !excludeList.isEmpty()) {
+            throw new ConfigException("Both include and exclude lists are specified. Please specify only one.");
+        }
+    }
+
+    @Override
+    public void close() {
+        cache.clear();
+    }
+
+    private enum Type {
+        ALL,
+        INCLUDE,
+        EXCLUDE
+    }
+
+    private Object getTimestampWithTimezone(String schemaName, Object fieldValue) {
+        Object updatedFieldValue = fieldValue;
+        ZoneId zoneId = ZoneId.of(convertedTimezone);
+        switch (schemaName) {
+            case ZonedTimestamp.SCHEMA_NAME:
+                ZonedDateTime zdt = ZonedDateTime.parse((String) fieldValue);
+                ZonedDateTime zdtWithZone = zdt.withZoneSameInstant(zoneId);
+                updatedFieldValue = ZonedTimestamp.toIsoString(zdtWithZone, null, 0);
+                break;
+            case ZonedTime.SCHEMA_NAME:
+                OffsetTime offsetTime = OffsetTime.parse((String) fieldValue, DateTimeFormatter.ISO_OFFSET_TIME);
+                OffsetTime offsetTimeWithZone = offsetTime.withOffsetSameInstant(zoneId.getRules().getOffset(Instant.now()));
+                updatedFieldValue = ZonedTime.toIsoString(offsetTimeWithZone, null);
+                break;
+            case MicroTimestamp.SCHEMA_NAME:
+                Instant microInstant = Instant.ofEpochMilli((long) fieldValue / 1_000);
```

Comment:
Precision can be lost, please use two parameter `ofEpochMilli` to provide micros and nanos.